d8/d8batch/README.md

```diff
@@ -0,0 +1,11 @@
+### D8 Batch
+
+#### tl;dr;
+Drupal's [Batch API](https://api.drupal.org/api/drupal/core!includes!form.inc/group/batch/8) did not change anything in Drupal 8, that is why this demo module also contains other useful examples from the Drupal 8 new features.
+
+#### About this module
+This module basically a demonstration of how can you import feed sources from a CSV file to the Drupal. The imported feed sources will be stored in nodes.
+
+The module also holds the definition of the necessary custom node type (Feed source) and its link field (field_feed_source_link) thanks to the new configuration management system.
```

Comment:
You should provide detailed description of each such feature. Eg. what are the parts needed if I only want to have a custom content type provided by my module?